JS/Script.js

```diff
@@ -1,73 +1,93 @@
 const bookCollection = document.querySelector('#collection');
 const addBook = document.querySelector('#add-book');
-
+let library;
 if (localStorage.getItem('Awesome Book Collection') === null) {
-  var library = [];
+  library = [];
 } else {
-  var library = JSON.parse(localStorage.getItem('Awesome Book Collection'))
+  library = JSON.parse(localStorage.getItem('Awesome Book Collection'));
 }
 
 function storeEntryData(titleElement, authorElement) {
-  let entryObject = {};
+  const entryObject = {};
   entryObject.title = titleElement.value;
   entryObject.author = authorElement.value;
   library.push(entryObject);
 }
```

Comment:
- [ OPTIONAL] The `push` method mutates arrays when used. Mutations may lead to unexpected and hard-to-debug issues, where data becomes incorrect somewhere, and you have no idea where it happens. I would advise you to use the `concat` method